src/features/access/RequestInfos.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+import { Grid, makeStyles, Typography } from '@material-ui/core'
+import moment from 'moment'
+
+const mock = {
+  author: 'Dr Marine Dijoux',
+  date: '03/09/2021 17:08',
+  perimeterAccess: ['Neurologie'],
+  comment:
+    'Ardeo, mihi credite, Patres conscripti (id quod vosmet de me existimatis et facitis ipsi) incredibili quodam amore patriae, qui me amor et subvenire olim impendentibus periculis maximis cum dimicatione capitis, et rursum, cum omnia tela undique esse intenta in patriam viderem, subire coegit atque excipere unum pro universis. Hic me meus in rem publicam animus pristinus ac perennis cum C. Caesare reducit, reconciliat, restituit in gratiam.'
+}
```

Comment:
this will be fetched from the back, can you setup the action/reducer and return this constant instead of making an api call ? this way we can the a prop type for this component